DapperDoodle.Tests/CaseHelperTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using NExpect;
+using static NExpect.Expectations;
+
+using NUnit.Framework;
+
+namespace DapperDoodle.Tests
+{
+    [TestFixture]
+    public class CaseHelperTests
+    {
+        [TestCase("TheQuickBrownFoxJumps")]
+        public void CaseKebabCase_DoesConvert_PascalCaseToKebabCase(string text)
+        {
+            Expect(text.ConvertCase(Case.KebabCase)).To.Equal("the-quick-brown-fox-jumps");
+        }
+
+        [TestCase("TheQuickBrownFoxJumps")]
+        public void PascalCase_DoesConvert_ToCamelCase(string text)
+        {
+            Expect(text.ConvertCase(Case.CamelCase)).To.Equal("theQuickBrownFoxJumps");
+        }
+
+        [TestCase("the_quick_brown_fox")]
+        public void SnakeCase_DoesConvert_ToCamelCase(string text)
+        {
+            Expect(text.ConvertCase(Case.CamelCase)).To.Equal("theQuickBrownFox");
+        }
+
+        [TestCase("TheQuickBrownFoxJumps")]
+        public void SnakeCase_DoesConvert_ToKebabCase(string text)
+        {
+            Expect(text.ConvertCase(Case.KebabCase)).To.Equal("the-quick-brown-fox-jumps");
+        }
+
+        [TestCase("theQuickBrownFoxJumps")]
+        public void KebabCase_DoesConvert_ToPascalCase(string text)
+        {
+            Expect(text.ConvertCase(Case.PascalCase)).To.Equal("Thequickbrownfoxjumps");
```

Comment:
I would have expected this to end up with the result "TheQuickBrownFoxJumps"; tho if you're using this for dapper mapping, if memory serves, casing doesn't matter that much (or perhaps that's just how we set up Dapper? at any rate, that would make the code simpler if you can just set up Dapper that way)

Reply:
(also note that there are analogous methods in PeanutButter.Utils as extensions on strings, ie `ToKebabCase`, `ToTitleCase`, 'ToCamelCase`)

Reply:
Thanks @fluffynuts I will have a look at those PeanutButter features.. I did look at those configurations for dapper. But when I tried to use it on Ubuntu I get errors. It seem on linux if the table names do not exactly match the SQL query it just basically says "The table doesn't exist." On Windows I never encountered this issue. That's why I thought I would include these case conversion methods.